packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -243,10 +243,13 @@ async function saveSnapshot(name: string, fetchedData: IFetchedData[], saveDir:
             console.error(`ERROR: Unable to get data for blob ${item.blobId}`);
             return;
         }
-        const data = bufferToString(buffer,"base64");
 
         if (!isFetchedTree(item)) {
-            fs.writeFileSync(`${outDir}/${item.filename}`, data);
+            // Just write the data as is.
+            fs.writeFileSync(`${outDir}/${item.filename}`, IsoBuffer.from(buffer));
```

Comment:
Ideally IsoBuffer should not be exported at all, and instead we use things like bufferFromString() & stringFromBuffer()
Here, I assume writeFileSync needs Buffer, not ArrayBuffer. If so, should we use it directly?

Reply:
Yeah... I originally was using Buffer.  this is neither bufferFromString() & stringFromBuffer(), because no conversion is needed.  All it need so wrap the ArrayBufferLike into a Buffer so that the File I/O API can treat it as a binary blob.   And I leded on IsoBuffer so that we can universally not use Buffer directly. 

Since this a node API, I can just switch it to us Buffer.